packages/code-du-travail-data/indexing/breadcrumbs.js

```diff
@@ -50,15 +50,22 @@ export function buildGetBreadcrumbs(themes) {
     return mainBreadcrumb || [];
   }
 
-  return function getBreadcrumbs(cdtnId) {
+  return function getBreadcrumbs(cdtnId, extras) {
```

Comment:
I hesitated between adding a new parameter or giving the whole document. Adding an extra argument seems like a minor change to the script (and to avoid breaking anything). Also, I prefer to give the required data as a parameter between a large object with a lot of unnecessary data.